lib/rambulance/exceptions_app.rb

```diff
@@ -66,7 +66,7 @@ def process_action(*)
       end
 
       # The #format method needs to be called after the sanitization above.
-      request.formats = request.formats.map(&:ref) << :html
+      request.formats << Mime::Type.register("text/html", :html)
```

Comment:
I should've replied earlier but we probably don't want to do this since `Mime::Type.register` overrides a global config. You could instead use `:text` (assuming it has a legit content type) since what's really important here is to not allow the request to fall into the limbo of rack middleware.